scm_v3c/radio.c

```diff
@@ -226,10 +226,6 @@ void repeat_rx_tx(repeat_rx_tx_params_t repeat_rx_tx_params) {
     uint8_t cfg_coarse_stop;
     uint8_t cfg_mid_stop;
     uint8_t cfg_fine_stop;
-		
-		// 1.1V (helps reorder assembly code)
-		uint8_t *txPacket_fix = repeat_rx_tx_params.txPacket;
```

Comment:
Where do these lines end up? Did they get moved?

Reply:
Everything from My last commit attempt is being removed and will be placed back in after doing a clang-format on the files.